frontend/packages/console-dynamic-plugin-sdk/src/app/components/status/Status.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import {
+  BanIcon,
+  ClipboardListIcon,
+  ExclamationTriangleIcon,
+  HourglassHalfIcon,
+  HourglassStartIcon,
+  NotStartedIcon,
+  SyncAltIcon,
+  UnknownIcon,
+} from '@patternfly/react-icons';
+import { StatusComponentProps } from '../../../extensions/console-types';
+import { DASH } from '../../constants';
+import { YellowExclamationTriangleIcon } from './icons';
+import { ErrorStatus, InfoStatus, ProgressStatus, SuccessStatus } from './statuses';
+import StatusIconAndText from './StatusIconAndText';
+
+type StatusProps = StatusComponentProps & {
+  status: string;
+};
+
+const Status: React.FC<StatusProps> = ({
```

Comment:
I believe we are not exposing these components (status/StatusIconAndText etc) from SDK via `api` as of now as we would only export `ListPageHeader` so it's fine but it would be nice to add some jsdocs for these if we plan to expose

https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts